test/UnitTests/Microsoft.Testing.Platform.UnitTests/CommandLine/CommandLineTests.cs

```diff
@@ -10,65 +10,125 @@ namespace Microsoft.Testing.Platform.UnitTests;
+        yield return (14, ["--option1", "a"], null, new CommandLineParseResultWrapper(null, new List<OptionRecord> { new("option1", ["a"]) }.ToArray(), []));
+        yield return (15, ["--option1:a"], null, new CommandLineParseResultWrapper(null, new List<OptionRecord> { new("option1", ["a"]) }.ToArray(), []));
+        yield return (16, ["--option1=a"], null, new CommandLineParseResultWrapper(null, new List<OptionRecord> { new("option1", ["a"]) }.ToArray(), []));
+        yield return (17, ["--option1=a", "--option1=b"], null, new CommandLineParseResultWrapper(null, new List<OptionRecord> { new("option1", ["a"]), new("option1", ["b"]) }.ToArray(), []));
+        yield return (18, ["--option1=a", "--option1 b"], null, new CommandLineParseResultWrapper(null, new List<OptionRecord> { new("option1", ["a"]), new("option1", ["b"]) }.ToArray(), []));
+        yield return (19, ["--option1=a=a"], null, new CommandLineParseResultWrapper(null, new List<OptionRecord> { new("option1", ["a=a"]) }.ToArray(), []));
+        yield return (20, ["--option1=a:a"], null, new CommandLineParseResultWrapper(null, new List<OptionRecord> { new("option1", ["a:a"]) }.ToArray(), []));
+        yield return (21, ["--option1:a=a"], null, new CommandLineParseResultWrapper(null, new List<OptionRecord> { new("option1", ["a=a"]) }.ToArray(), []));
+        yield return (22, ["--option1:a:a"], null, new CommandLineParseResultWrapper(null, new List<OptionRecord> { new("option1", ["a:a"]) }.ToArray(), []));
+        yield return (23, ["--option1:a:a", "--option1:a=a"], null, new CommandLineParseResultWrapper(null, new List<OptionRecord> { new("option1", ["a:a"]), new("option1", ["a=a"]) }.ToArray(), []));
+        yield return (24, ["--option1", "a", "@test.rsp", "--option5", "e"], [("test.rsp",
+            """
+            --option2 b
+            --option3 c
+            --option4 d
+            """)], new CommandLineParseResultWrapper(null, new List<OptionRecord>
+        {
+            new("option1", ["a"]),
+            new("option2", ["b"]),
+            new("option3", ["c"]),
+            new("option4", ["d"]),
+            new("option5", ["e"]),
+        }.ToArray(), []));
+        yield return (25, ["--option1", "a", "@test1.rsp", "--option6", "f"], [
+            ("test1.rsp",
+            """
+            --option2 b
+            @test2.rsp
+            --option5 e
+            """),
+            ("test2.rsp",
+            """
+            --option3 c
+            --option4 d
+            """)], new CommandLineParseResultWrapper(null, new List<OptionRecord>
+        {
+            new("option1", ["a"]),
+            new("option2", ["b"]),
+            new("option3", ["c"]),
+            new("option4", ["d"]),
+            new("option5", ["e"]),
+            new("option6", ["f"]),
+        }.ToArray(), []));
```

Comment:
pls add a pair of more sample with 
1) rsp file at the start 
2) rsp file t the end
3) only rsp file
4) 2 rsp file only

Reply:
Done